components/popup/src/vwc-popup-base.ts

```diff
@@ -0,0 +1,198 @@
+import {
```

Comment:
@yinonov - have we given up on `base` suffix?

Reply:
hmm in this repo it's still `base` all around. maybe we shouldn't go into trouble replacing all files in this codebase